data/strings/types_strings.txt

```diff
@@ -26975,7 +26978,7 @@
   [type.piste_type.downhill.expert]
     ref = type.piste_type.downhill
     en = Expert Downhill Ski Run
-    de = Profi Abfahrt
+    de = Profi Ski Abfahrt
```

Comment:
Ski-Abfahrt

Reply:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Profi-Ski-Abfahrt
```